### PR TITLE
fix(Forms): use rem unit over px

### PR DESCRIPTION
**Pull request by tujoworker**

### CODE REVIEW

File: packages/dnb-eufemia/src/extensions/forms/Layout/style/dnb-card.scss [hunk 1]
```diff
@@ -1,9 +1,10 @@
 .dnb-forms-card {
   background-color: var(--color-white);
-  border: 4px solid #F2F2F5;
+  border: 4px solid #f2f2f5;
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/Layout/style/dnb-card.scss:3]:
```suggestion
  border: 4px solid var(--color-lavender);
```

File: packages/dnb-eufemia/src/extensions/forms/utils/TestElement/style/dnb-test-element.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .dnb-forms-test-element {
   border: 1px dashed #808080;
-  border-radius: 6px;
+  border-radius: 0.5rem;
```
Comment (langz) [inline packages/dnb-eufemia/src/extensions/forms/utils/TestElement/style/dnb-test-element.scss:3]:
```suggestion
  border-radius: 0.375rem;
```

**Review by langz — APPROVED** [review]

File: packages/dnb-eufemia/src/extensions/forms/Layout/style/dnb-card.scss (hunk 1 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/extensions/forms/Layout/style/dnb-card.scss:3]: This is taken care of in this PR: https://github.com/dnbexperience/eufemia/pull/2588/commits/4fcdb1abef76d0f2fc7711aed942d56c214c9dc9 – my bad that it got "corrected" by prettier :D

File: packages/dnb-eufemia/src/extensions/forms/utils/TestElement/style/dnb-test-element.scss (hunk 2 above)
Comment (tujoworker) [reply packages/dnb-eufemia/src/extensions/forms/utils/TestElement/style/dnb-test-element.scss:3]: I don't think it does matter, does it?

**Pull request merged**